dev.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: turbograft
+up:
+- ruby: 2.3.1
+- bundler
+commands:
+  test: bundle exec rake test
```

Comment:
`bundle exec rake` is better I because it will always run in the context of your current project's bundle. If you have system gems `rake` could use those instead.